include/TPP/Passes.h

```diff
@@ -83,6 +83,8 @@ std::unique_ptr<OperationPass<func::FuncOp>> createConvertLinalgToTppPass();
 std::unique_ptr<OperationPass<func::FuncOp>>
 createConvertLinalgToTppPass(bool, bool, ArrayRef<int64_t> tiles = {});
 std::unique_ptr<OperationPass<func::FuncOp>> createConvertTppToLoopsPass();
+std::unique_ptr<OperationPass<func::FuncOp>>
```

Comment:
Why a new one? Perhaps just adding the `parallel` flag (default false) would do?

Reply:
The defaults should have only one origin and that is the tablegen. Adding default argument values at the header only adds maintenance burden as now you have to sync both .td and .h.
(guilty of that, I'll have to clean it up one day 😅 )

Reply:
Then why not add the option/default to the tablegen?

Reply:
It is there but I still need a constructor that allows me to pass a custom value.

Reply:
Discussed offline, let's keep it consistent with our other passes.
TODO: cleanup double API, add a comment to the header